airbyte-integrations/connectors/source-freshdesk/source_freshdesk/spec.json

```diff
@@ -22,6 +22,13 @@
         "title": "Requests per minute",
         "type": "integer",
         "description": "Number of requests per minute that this source allowed to use."
+      },
+      "start_date": {
+        "title": "Tickets updated since",
```

Comment:
So this option is only used for tickets? If so, the title should be `Tickets lookback window` and param name `tickets_lookback_window`